### DISCUSSION TITLE
Is WASM+SIMD relevant for manifold performance ?

### DISCUSSION THREAD

**Discussion by hrgdavor** [elalish/manifold#278] [Q&A]
Looks like SIMD is possible for Chrome and Edge in WASM , maybe even FF (Firefox 89 for x64, x86, Firefox 90 for arm64.)
https://v8.dev/features/simd

My knowledge here is extremely limited, but I just wanted point this out if devs will find it interesting, If it could bring some perf gain to build SIMD enabled version of Manifold.

**Comment by pca006132** [comment]
I think we probably won't write SIMD ourselves, but enabling WASM+SIMD may be beneficial in some cases, especially if we can turn the sorts to https://github.com/google/highway/tree/master/hwy/contrib/sort.
